sdk/search/search-documents/CHANGELOG.md

```diff
@@ -9,13 +9,17 @@
 ### Breaking Changes
 
 - Update `SearchIndexerDataNoneIdentity.odatatype`. [#23495](https://github.com/Azure/azure-sdk-for-js/pull/23495)
+- Fix `SearchClient.getDocument` with `selectedFields` option returning a wide result with `undefined` non-optional fields [#23627](https://github.com/Azure/azure-sdk-for-js/pull/23627)
 
 ### Bugs Fixed
 
+- Allow nested field names to be specified in `select` and `searchField` options [#23627](https://github.com/Azure/azure-sdk-for-js/pull/23627)
```

Comment:
It'll be helpful to include a quick list of which options bag types this affects.

Reply:
Done!